BackofficeBundle/StrategyManager/GenerateFormManager.php

```diff
@@ -57,6 +57,25 @@ public function getDefaultConfiguration(BlockInterface $block)
     }
 
     /**
+     * Get the required Uri parameters for the block
+     *
+     * @param BlockInterface $block
+     *
+     * @return array
+     */
+    public function getRequiredUriParameter(BlockInterface $block)
+    {
+        /** @var GenerateFormInterface $strategy */
+        foreach ($this->strategies as $strategy) {
+            if ($strategy->support($block)) {
+                return $strategy->getRequiredUriParameter();
+            }
+        }
+
+        return array();
```

Comment:
Why not, but this way is used in getDEfaultParameter. So we should change it too or keep it like that, @nicolasThal